docs/REST API/rest-api.md

```diff
@@ -4872,7 +4873,7 @@ List all configuration profiles for macOS and Windows hosts enrolled to Fleet's
       "created_at": "2023-04-31T00:00:00Z",
       "updated_at": "2023-04-31T00:00:00Z",
       "checksum": "aCLemVr)",
-      "labels": [
+      "labels_include_all": [
```

Comment:
Break the existing `labels` key in the response. Change it to `labels_include_all`

Reply:
Note that we added a second name for `GET /api/v1/fleet/mdm/profiles`: `GET /api/v1/fleet/configuration_profiles`

These changes should apply if using either endpoint.

Open docs PR for the new name is here: https://github.com/fleetdm/fleet/pull/16728